ta/pkcs11/src/pkcs11_attributes.c

```diff
@@ -68,6 +68,7 @@ check_mechanism_against_processing(struct pkcs11_session *session,
 	switch (step) {
 	case PKCS11_FUNC_STEP_INIT:
 		switch (function) {
+		case PKCS11_FUNCTION_DIGEST:
```

Comment:
I don't think we should add it here. For Digest function, mechanism_supported_flags() at line85 below should be called.

Reply:
right. fixing.